arch/x86/include/asm/bitops.h

```diff
@@ -166,7 +166,11 @@ static inline void __clear_bit(long nr, volatile unsigned long *addr)
  */
 static inline void __clear_bit_unlock(long nr, volatile unsigned long *addr)
 {
+#ifndef CONFIG_KTSAN
 	barrier();
+#else /* CONFIG_KTSAN */
+	ktsan_thread_fence(ktsan_memory_order_release);
```

Comment:
replace it with:

static inline void __clear_bit_unlock(long nr, volatile unsigned long *addr) {
	ktsan_atomic_store(addr, *(unsigned long*)addr & ^(1<<nr), ktsan_memory_order_release);
}